docs/source/installation.rst

```diff
@@ -100,98 +100,270 @@ Step 1: Setup dedicated python environment
 
-                    .. code::
+                            and, with CUDA 11.1:
 
-                       pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html
+                            ::
 
-                .. tab:: CPU Support
+                                git clone https://github.com/ivadomed/ivadomed.git
 
-                    If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
+                                cd ivadomed
+
+                                pip install -e . --extra-index-url https://download.pytorch.org/whl/cu111
+
+
+                .. tab:: Contributor or Developer Installation
+
+                    To contribute to the project, we recommend installing ``ivadomed``
+                    from source along with additional dependencies related to building
+                    documentation and testing
 
                     .. tabs::
 
-                        .. tab:: Windows/Linux
+                        .. tab:: Linux
+
+                            with CUDA 10.2:
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .[dev]
+
+                            with CUDA 11.1:
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .[dev] --extra-index-url https://download.pytorch.org/whl/cu111
+
+                        .. tab:: Windows
+
+                            with CUDA 10.2:
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .[dev] --extra-index-url https://download.pytorch.org/whl/cu102
 
-                            .. code::
+                        
+                            with CUDA 11.1:
 
-                               pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .[dev] --extra-index-url https://download.pytorch.org/whl/cu111
+
+
+        .. tab:: CPU Support
+
+            .. tabs:: 
+                
+                .. tab:: Package Installation (Recommended)
+
+                    .. tabs::
+
+                        .. tab:: Linux
+
+                            ::
+
+                                pip install ivadomed --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
I'm personally invested in this one line, because we [use it in SCT](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3790), but maybe it's better to drop it. The torch people think linux people should waste their space, so let's just roll with that?

Alternately, this could fit in a `tabs:::` above, next to the GPU section, maybe something like:

> Installation | CPU-based Installation | GPU-based Installation (Windows) | Ampere GPU-based Installation

So the 80% case is the first tab, which wastes some Linux users' space who haven't read closely but that's not the end of the world, the 15% case is the people who read the CPU instructions, another say ~5% of people are bold explorers using GPUs on Windows, and the <1% case is the people rich enough to have Ampere GPUs.

And the only difference between each tab is

a. Instructing people how to decide if that case applies to them
b. Giving different `extra-index-url` lines for their `pip.conf`

Reply:
Flattening the tab hierarchy would result in repetition of some GPU instructions.